pkg/aks/components/CruAks.vue

```diff
@@ -596,6 +596,60 @@ export default defineComponent({
       ];
     },
 
+    // in the labeledselect, networks will be shown as 'groups' with their subnets as selectable options
+    // it is possible for a virtual network to have no subnets defined - they will be excluded from this list
+    virtualNetworkOptions() {
+      const out: {label: string, kind?: string, disabled?: boolean, value?: string, virtualNetwork?: any, key?: string}[] = [{ label: this.t('generic.none') }];
+
+      if (this.loadingVirtualNetworks) {
+        return out;
+      }
+      (this.allVirtualNetworks || []).forEach((network) => {
```

Comment:
You probably don't need the fallback since you have a default value in the data block.

Reply:
You're right: it's always defined. I've removed this fallback.